src/cli/commands/upgrade-interactive.js

```diff
@@ -57,26 +59,15 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
   const headerPadding = (header, key) =>
     `${reporter.format.bold.underline(header)}${' '.repeat(maxLengthArr[key] - header.length)}`;
 
-  const colorizeName = ({current, wanted}) => (current === wanted ? reporter.format.yellow : reporter.format.red);
+  const colorizeName = (from, to) => reporter.format[colorKeyFromVersions(from, to)];
```

Comment:
The reason that I changed this not to use destructuring was to keep it pure as the second argument could be either `wanted` or `latest` if the `--latest` flag is set.